include/openrave/planner.h

```diff
@@ -487,15 +487,20 @@ class OPENRAVE_API PlannerStatus
 {
 public:
     PlannerStatus();
-    PlannerStatus(const int statusCode);
-    PlannerStatus(const std::string& description, const int statusCode, CollisionReportPtr report=CollisionReportPtr());
-    PlannerStatus(const std::string& description, const int statusCode, const IkParameterization& ikparam, CollisionReportPtr report=CollisionReportPtr());
-    PlannerStatus(const std::string& description, const int statusCode, const std::vector<dReal>& jointValues, CollisionReportPtr report=CollisionReportPtr());
+    PlannerStatus(const std::string& description, const int statusCode);
+
+    // PlannerStatus();
+    // PlannerStatus(const int statusCode);
+    // PlannerStatus(const std::string& description, const int statusCode, CollisionReportPtr report=CollisionReportPtr());
+    // PlannerStatus(const std::string& description, const int statusCode, const IkParameterization& ikparam, CollisionReportPtr report=CollisionReportPtr());
+    // PlannerStatus(const std::string& description, const int statusCode, const std::vector<dReal>& jointValues, CollisionReportPtr report=CollisionReportPtr());
+    
```

Comment:
Please keep all the constructors exactly same as before. I don't think you have a good enough reason to change them.

Reply:
I `grep`ed through the projects, and it seems like these constructors are not being used. Also, part of the motivation is because I changed the member variables.

Reply:
I `grep`ed through the projects too, and I see plenty of usages. Please compile openrave before you ask for review.